src/site/content/en/blog/embed-best-practices/index.md

```diff
@@ -0,0 +1,314 @@
+
+    ```html
+    <lite-youtube videoid="ogfYd705cRs" playlabel="Play: Keynote (Google I/O '18)"></lite-youtube>
+    ```
+	Following is a comparison between the lite-youtube-embed and the actual embed.
+
+	<div class="w-table-wrapper">
+    <table>
+    <tbody>
+      <tr>
+        <td>{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/EcTxjLs9SUb1ofALN8rA.png", alt="Lite YouTube embed", width="800", height="521" %}</td>
+        <td>{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/cYG1NJqM8ZoLkYOi6xFJ.png", alt="Actual YouTube embed", width="800", height="502" %}</td>
+      </tr>
+    </tbody>
+    <caption>YouTube click-to-load</caption>
+    </table>
+    </div>
+
+    Similar other facades also available for YouTube and Vimeo players are [lite-youtube](https://github.com/justinribeiro/lite-youtube), [lite-vimeo-embed](https://github.com/luwes/lite-vimeo-embed), and [lite-vimeo](https://github.com/slightlyoff/lite-vimeo).
+
+* Chat widget facade
+
+    [React-live-chat-loader](https://github.com/calibreapp/react-live-chat-loader) loads a button that looks like a chat embed instead of the embed itself. It can be used with various chat provider platforms such as Intercom, Help Scout, Messenger, and so on. The look-alike widget is much lighter than the chat-widget and loads faster. It can be replaced by the actual chat widget when the user hovers or clicks on the button or if the page has been idle for a long time. The [Postmark case study](https://wildbit.com/blog/2020/09/30/getting-postmark-lighthouse-performance-score-to-100) explains how they implemented react-live-chat-loader and performance improvements they achieved.
+
+   {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/XyJON43TV8h1qWNZV1Ev.png", alt="Postmark chat widget", width="800", height="389" %}
+
+### **Remove or replace embeds with links**
+
+If you find that some third-party embeds result in poor loading performance and using any of the techniques above is not an option, the simplest thing that you can do is remove the embed entirely. If you still want your users to be able to access the content in the embed, you can provide a link to it with `target="_blank"` so that the user can click and view it in another tab. 
+
+
+## Layout stability
+
+While dynamically loading embedded content can improve the loading performance of a page, it can sometimes cause unexpected movement of page content. This is known as layout shift. 
+
+Since visual stability is important to guarantee a smooth user experience, [Cumulative Layout Shift (CLS)](https://web.dev/cls/) measures how often those shifts happen and how disruptive they are.
```

Comment:
```suggestion
Since visual stability is important to guarantee a smooth user experience, [Cumulative Layout Shift (CLS)](/cls/) measures how often those shifts happen and how disruptive they are.
```